server/schemas/resolverRoutes/queries.js

```diff
@@ -1,13 +1,43 @@
 const { AuthenticationError } = require('apollo-server-express');
 const { User, Brand, Skater, SkateVideo } = require('../../models');
+const { eventNames } = require('../../models/Soundtrack');
 
 const Query = {
-        me: async (parent, args, context) => {
-          if (context.user) {
-            return User.findOne({ _id: context.user._id });
-          }
-          throw new AuthenticationError('You need to be logged in!');
-        },
+  me: async (parent, args, context) => {
+    if (context.user) {
+      return User.findOne({ _id: context.user._id });
+      }
+      throw new AuthenticationError('You need to be logged in!');
+  },
+  brand: async ( parent , { _id }) => {
+    return await Brand.findById(_id).populate('skateVideos')
+  },
+  brands: async () => {
+    return await Brand.find().populate('skateVideos');
```

Comment:
Don't you need to send in an empty object with these queries?

Reply:
I was looking through examples and I do not believe you need an empty object

Reply:
Ok